src/parser.rs

```diff
@@ -302,8 +311,15 @@ fn string_literal(input: &str) -> IResult<&str, CypherValue> {
     )(input)
 }
 
+fn boolean_literal(input: &str) -> IResult<&str, CypherValue> {
+  alt((
+      map(tag("false"), |_| CypherValue::Boolean(false)),
+      map(tag("true"), |_| CypherValue::Boolean(true))
```

Comment:
maybe we could use `nom::tag_no_case` instead of `tag`?

Reply:
Yes, of course.